packages/sdk/browser-wallets.d.ts

```diff
@@ -0,0 +1 @@
+export * from "./dist/browser-wallets/index";
```

Comment:
Is this auto generated by vite-plugin-dts? Do we need to commit these?

Reply:
These are not auto generated. We will have to commit these when adding new entry points, otherwise there will be no type information.